lib/tasks/dev.rake

```diff
@@ -2,12 +2,9 @@ if Rails.env.development? || Rails.env.staging?
   namespace :dev do
     desc "Seed data for development environment"
     task prime: "db:setup" do
-      User.create(
-        first_name: "John",
-        last_name: "Wayne",
-        email: "john@example.org",
-        password: "stagecoach"
-      )
+      include FactoryBot::Syntax::Methods
+
+      create(:user, first_name: "John", last_name: "Wayne")
```

Comment:
Not really a fan of using factories outside of tests.

The point of factories is to use "inheritance" and "composition" to ease setup data where we only want some attributes for testing purposes but we still need to fill the rest to pass the validations.

In this task, we want real-world-like data, where all attributes matter, created with the same ActiveRecord statements the controllers, models, and services are using.

It also creates an unnecessary coupling with an external dependency.

Reply:
I agree, although it doesn't seem like a blocking issue. It was following thoughtbot's examples in their [suspenders](https://github.com/thoughtbot/suspenders/blob/master/templates/dev.rake) repo.